.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Generate release manifests
+        run: make release-manifests
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Upload generated release manifests
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: _out/vm-console-proxy.yaml
+          asset_name: vm-console-proxy.yaml
+          asset_content_type: text/plain
+
+      - name: Upload to Quay
```

Comment:
nit: Can you move this step above the `Upload generated release manifests`. That way, if image build and upload fails for any reason, we will notice it easily, because manifests will not be added to the release.

Reply:
Yep, done.